phpunit.xml.dist

```diff
@@ -7,12 +7,11 @@
 
   <php>
     <const name="PHPUNIT_TESTSUITE" value="true"/>
-    <const name="PHPUNIT_TESTSUITE_EXTENSION_SELENIUM_HOST" value="127.0.0.1"/>
+    <!--<const name="PHPUNIT_TESTSUITE_EXTENSION_SELENIUM_HOST" value="127.0.0.1"/>-->
     <const name="PHPUNIT_TESTSUITE_EXTENSION_SELENIUM_PORT" value="4444"/>
-    <const name="PHPUNIT_TESTSUITE_EXTENSION_SELENIUM_BROWSER" value="*firefox"/>
-    <const name="PHPUNIT_TESTSUITE_EXTENSION_SELENIUM2_BROWSER" value="firefox"/>
+    <const name="PHPUNIT_TESTSUITE_EXTENSION_SELENIUM_BROWSER" value="chrome"/>
+    <const name="PHPUNIT_TESTSUITE_EXTENSION_SELENIUM_BROWSER_VERSION" value="74.0"/>
```

Comment:
ops. All of these should be `<env>` so they can be overriden by env vars

Reply:
I agree environment variables would be better, but some tests (e.g. `Extensions_Selenium2TestCaseMultipleBrowsersPropertyTest`) use these constants in their property declaration. Unfortunately you can't use `getenv` or `$_ENV` there. A workaround would be to use envrionment variables in `phpunit.xml.dist` and define all constants based on those variables in `phpunit-selenium-bootstrap.php`, but in my opinion that's not a really nice solution.